bungee/src/main/java/me/leoko/advancedban/bungee/utils/CloudNetCloudPermsOfflineUser.java

```diff
@@ -0,0 +1,20 @@
+package me.leoko.advancedban.bungee.utils;
+
+import me.leoko.advancedban.utils.Permissionable;
+import de.dytanic.cloudnet.driver.CloudNetDriver;
+import de.dytanic.cloudnet.driver.permission.IPermissionUser;
+
+import java.util.UUID;
+
+public class CloudNetCloudPermsOfflineUser implements Permissionable {
+    private IPermissionUser permissionUser;
+
+    public CloudNetCloudPermsOfflineUser(String name) {
+        this.permissionUser = CloudNetDriver.getInstance().getPermissionManagement().getUsers(name).get(0);
```

Comment:
An ArrayList throws an IndexOutOfBounds exception if you try to access an invalid index (if there is no player with that name) instead of returning null which is what you would need for the check below to work properly.